contracts/voting/dao-voting-cw4/src/contract.rs

```diff
@@ -164,10 +238,28 @@ pub fn query_total_power_at_height(deps: Deps, env: Env, height: Option<u64>) ->
 }
 
 pub fn query_info(deps: Deps) -> StdResult<Binary> {
-    let info = cw2::get_contract_version(deps.storage)?;
+    let info = get_contract_version(deps.storage)?;
     to_json_binary(&dao_interface::voting::InfoResponse { info })
 }
 
+pub fn query_is_active(deps: Deps) -> StdResult<Binary> {
+    let active_threshold = ACTIVE_THRESHOLD.load(deps.storage)?;
+    let group_contract = GROUP_CONTRACT.load(deps.storage)?;
+    let total_weight: TotalWeightResponse = deps.querier.query_wasm_smart(
+        group_contract,
+        &cw4_group::msg::QueryMsg::TotalWeight { at_height: None },
+    )?;
+
+    let is_active = match active_threshold {
+        ActiveThreshold::AbsoluteCount { count } => {
+            total_weight.weight >= count.to_string().parse::<u64>().unwrap()
```

Comment:
So the active threshold `count` here is `Uint128`. Rather than doing this weird `count.to_string().parse::<u64>().unwrap()` (converting to string and then parsing that into `u64`). It's cleaner to just convert the `total_weight.weight` `u64` into `Uint128`. Don't loose any fidelity that way either (though unlike will be using very large numbers in this context).

Something like:
```suggestion
            Uint128::new(total_weight.weight as u128) >= count
```

As a bonus, this gets rid of an `unwrap()` which we generally don't like to use in smart contracts.